content/docs/buildpacks/language-family-buildpacks/dotnet-core.md

```diff
@@ -1,283 +0,0 @@
----
-title: ".Net Core Buildpack"
-weight: 301
```

Comment:
My only concern with this is that some of this metadata is very website specific and relative to the other docs pages. It seems like we need a better way to organize the docs sidebar if we move to hugo modules.

Reply:
Why would moving to hugo modules impact the sidebar?

Reply:
This might not be a big issue, but it is just confusing to me that the organization of where the doc appears in the sidebar, it's parent, etc. is controlled by metadata (`weight` and `parent`) that live outside of the website repo. Does that make sense?

Reply:
As far as hugo is concerned at deploy time, the mounted file is exactly in the location of this removed file. So any fancy stuff that hugo does should be preserved.

Reply:
Ahh got it, I misunderstood how the modules work. Thanks for clarifying.